apps/src/gamelab/GameLab.js

```diff
@@ -238,13 +238,12 @@ GameLab.prototype.init = function (config) {
  */
 GameLab.prototype.setupReduxSubscribers = function (store) {
   var state = {};
-  var boundOnIsRunningChange = this.onIsRunningChange.bind(this);
-  store.subscribe(function () {
+  store.subscribe(() => {
     var lastState = state;
     state = store.getState();
 
     if (!lastState.runState || state.runState.isRunning !== lastState.runState.isRunning) {
-      boundOnIsRunningChange(state.runState.isRunning);
+      this.onIsRunningChange(state.runState.isRunning);
```

Comment:
This bit of cleanup looks unrelated - I just happened to touch this method in an intermediate state where Gamelab.js was involved in communication with the iframe.  See commit-by-commit changes if you're curious.